### PR TITLE
CSS: Improvements to the horizontal description list styling.

### PR DESCRIPTION
**Pull request by thomasgohard**
For #6812.

### CODE REVIEW

**Comment by hsrudnicki** [comment]
Just by occasion, can this issue be closed as well?

**Comment by LaurentGoderre** [comment]
Please don't nest media queries inside selector. If you need media query, create a new file and include it in the `_wet-boew.css`

**Comment by LaurentGoderre** [comment]
Where does the 768px come from?

**Comment by LaurentGoderre** [comment]
Ah nevermind. Bootstrap seems to be doing this a lot.

File: src/base/bootstrap-overrides/_base.scss [hunk 1]
```diff
@@ -45,8 +45,24 @@ a {
 }
 
 .dl-horizontal {
-	dt {
-		white-space: normal;
+	@media (min-width: 768px) {
```
Comment (pjackson28) [inline src/base/bootstrap-overrides/_base.scss]: Shouldn't we be using $screen-sm-min instead?

Comment (LaurentGoderre) [inline src/base/bootstrap-overrides/_base.scss]: Bootstrap uses this media query a lot in it's code. It seems like he just copied it.

Comment (pjackson28) [inline src/base/bootstrap-overrides/_base.scss]: Maybe so, but it is also defined in a variable so shouldn't we stick to the variables? If Bootstrap changes from 768px to 769px then they are responsible for updating all their variables and code but by us using a variable then we wouldn't have to make a change if that scenario were to happen.

Comment (LaurentGoderre) [inline src/base/bootstrap-overrides/_base.scss]: Aaaah good call, I only looked at the generated CSS. You are right!!

**Comment by andre-gh** [comment]
Can this pull request be closed, has it gone stale?

**Comment by andre-gh** [comment]
@thomasgohard Since this fixes an issue are you willing to resume this pull request or should I take it over?

**Comment by thomasgohard** [comment]
@andre-gh I've rebased the PR. It's ready for review and merging.

**Comment by andre-gh** [comment]
Thanks, so much much better aligned right.

**Comment by andre-gh** [comment]
Er, aligned correctly

**Pull request merged**